examples/routing_service/udp_socket_adapter/src/SocketConnection.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * (c) 2019 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ *
+ * RTI grants Licensee a license to use, modify, compile, and create derivative
+ * works of the Software.  Licensee has the right to distribute object form
+ * only for use with RTI products.  The Software is provided "as is", with no
+ * warranty of any type, including any warranty for fitness for any purpose.
+ * RTI is under no obligation to maintain or support the Software.  RTI shall
+ * not be liable for any incidental or consequential damages arising out of the
+ * use or inability to use the software.
+ */
+
+#ifndef SOCKETCONNECTION_HPP
+#define SOCKETCONNECTION_HPP
+
+#include <rti/routing/adapter/AdapterPlugin.hpp>
+#include <rti/routing/adapter/Connection.hpp>
+
+ #include "SocketInputDiscoveryStreamReader.hpp"
+
+class SocketConnection : public rti::routing::adapter::Connection {
```

Comment:
Add comment describing the class briefly and any details interesting for users to know.

Reply:
I added a comment describing this class